frontend/src/component/executiveDashboard/ExecutiveDashboard.tsx

```diff
@@ -18,6 +18,19 @@ const StyledGrid = styled(Box)(({ theme }) => ({
 export const ExecutiveDashboard: VFC = () => {
     const { executiveDashboardData, loading, error } = useExecutiveDashboard();
 
+    const calculateFlagPerUsers = () => {
```

Comment:
I was considering this calculation in the backend and decided it's ok to have it in frontend for now. What do you prefer?